requirements-dev.txt

```diff
@@ -3,6 +3,7 @@ moto
 coverage
 pyyaml
 requests_mock
+responses
```

Comment:
We have requests_mock and responses here. They both seem to be responsible for requests mocking. Are they both necessary?

Reply:
I removed requests_mock.